elements/rh-chip/rh-chip.ts

```diff
@@ -0,0 +1,111 @@
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { property } from 'lit/decorators/property.js';
+import { query } from 'lit/decorators/query.js';
+import { colorContextConsumer, type ColorTheme } from '../../lib/context/color/consumer.js';
+import { consume } from '@lit/context';
+
+import { context, type RhChipGroupContext } from './context.js';
+
+import '@rhds/elements/rh-icon/rh-icon.js';
+
+import styles from './rh-chip.css';
+
+export class ChipCheckedEvent extends CustomEvent<{ checked: boolean }> {
+  constructor(checked: boolean) {
+    super('chip-checked', {
+      bubbles: true,
+      composed: true,
+      detail: { checked },
+    });
+  }
+}
```

Comment:
```suggestion
export class ChipChangeEvent extends Event {
  constructor(public checked: boolean) {
    super('change', {
      bubbles: true,
      cancellable: true,
    });
  }
}
```

see accordion, tab, etc.

Reply:
Handled in [f23cbc3](https://github.com/RedHat-UX/red-hat-design-system/pull/2198/commits/f23cbc3570d556244212cb887ef86bfc53d9ed84).